awx/ui_next/src/screens/User/UserList/UserList.test.jsx

```diff
@@ -280,16 +261,8 @@ describe('UsersList without full permissions', () => {
     });
 
     wrapper = mountWithContexts(<UsersList />);
-    await waitForElement(
-      wrapper,
-      'UsersList',
-      el => el.state('hasContentLoading') === true
-    );
-    await waitForElement(
-      wrapper,
-      'UsersList',
-      el => el.state('hasContentLoading') === false
-    );
+    await waitForElement(wrapper, 'ContentLoading', el => el.length === 1);
```

Comment:
We _can_ check for the `ContentLoading` component in this test because we mock the api with an empty set of items, so that code path remains tested.